apps/judicial-system/web/src/routes/Shared/Requests/Requests.tsx

```diff
@@ -89,14 +89,14 @@ export const Requests: React.FC = () => {
       caseToDelete.state === CaseState.SUBMITTED ||
       caseToDelete.state === CaseState.RECEIVED
     ) {
-      const caseDeleted = await transitionCase(
-        caseToDelete,
-        CaseTransition.DELETE,
+      const notificationSent = await sendNotification(
+        caseToDelete.id,
+        NotificationType.REVOKED,
       )
 
-      if (caseDeleted) {
+      if (notificationSent) {
```

Comment:
Consider not checking for notification success. The notification is not critical and we should probably delete the case even if notifications fail.